roukmoute/hashids-bundle/1.4/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Roukmoute\\HashidsBundle": ["all"]
```

Comment:
Travis [complains here](https://travis-ci.org/symfony/recipes-contrib/jobs/341340179#L610). This line should be: 
`Roukmoute\\HashidsBundle\\RoukmouteHashidsBundle`